changelogs/unreleased/gh-7512-raft-leader-hang.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/replication
+
+* Fixed followers with `box.cfg.election_mode` turned on not noticing leader
+  hang due to some long request without yields (like `while true do end`)
```

Comment:
In commit message you mention more realistic case with long select/iterating without yields, maybe mention it in changelog instead of artificial `while true do end`.

Reply:
Sure:
```diff
diff --git a/changelogs/unreleased/gh-7512-raft-leader-hang.md b/changelogs/unreleased/gh-7512-raft-leader-hang.md
index 7ec834057..506180a23 100644
--- a/changelogs/unreleased/gh-7512-raft-leader-hang.md
+++ b/changelogs/unreleased/gh-7512-raft-leader-hang.md
@@ -1,5 +1,5 @@
 ## bugfix/replication
 
 * Fixed followers with `box.cfg.election_mode` turned on not noticing leader
-  hang due to some long request without yields (like `while true do end`)
-  (gh-7512).
+  hang due to some long request without yields, like a `select{}` from a large
+  space or `pairs` iteration without yields between loop cycles) (gh-7512).
```